MibTeX/src/de/mibtex/clean/BibtexCleaner.java

```diff
@@ -16,11 +16,12 @@
  * 
  */
 public class BibtexCleaner {
+	private static final String DOI = "doi";
 	private static List<String> attributesToRemove;
 
 	public static void main(String[] args) {
 		attributesToRemove = Arrays.asList(
-				  "doi",
+//				  DOI,
 				  "issn",
```

Comment:
?

Reply:
This should not be commented. Will remove the `//`.